src/utils/Endpoint.js

```diff
@@ -0,0 +1,41 @@
+import transformToHarRequest from './transformToHarRequest';
+import { HTTPSnippet } from 'httpsnippet';
+
+const supportedClients = {
+  curl: {
+    targetId: 'shell',
+    options: { short: true, indent: ' ', prettifyJson: true },
+  },
+};
+
+class Endpoint {
```

Comment:
@MeganSteenkamp Suggestion: Remove this class wrapper to reduce boilerplate. Just export a `create` function instead